### ISSUE TITLE
Allow reading config from file instead of CLI args

### ISSUE THREAD

**Issue by ObiWanRohan** [firediscordchat/peef#11]
Currently the API Key and Guild IDs are passed to the app through CLI args.
It would be more convenient for deployment and development if these values could be specified in a config or a `.env` file.

The `viper` package seems like a good way to go about it.

**Comment by cnpryer** [comment]
Yup I agree. This should be the direction we head in. I’m not opposed to keeping certain things as optional ways to spin up the bot (use arts if you want, otherwise expect environment variables).

I believe `API_KEY` is currently utilized like this. I haven’t implemented `viper` yet but it looks like a good idea as well. Right now I used `os.Getenv()`.

**Comment by cnpryer** [comment]
I'm not sure if your branch contains the `viper` implementation, but I figured I'd check this out and branch to [`viper`](https://github.com/firediscordchat/peef/tree/viper).
